### ISSUE TITLE
Guid format error: "The format is invalid"

### ISSUE THREAD

**Issue by ssaylanc** [pauldemarco/flutter_blue#105]
For scanning devices with a UUID like "FFF0", I receive "The format is invalid" error.

**Comment by Fleximex** [comment]
I ran into a similar problem. It seems FlutterBlue uses full UUID's.
So for example I used UUID "FFE1" in a totally different Bluetooth C# framework. But FlutterBlue requires "0000ffe1-0000-1000-8000-00805f9b34fb" in my case.

A good practice in understanding UUID's, services, descriptors, etc is downloading the app 'BLE Scanner' for either Android or iOS. The app will give you all the information of the Bluetooth device in question.
https://play.google.com/store/apps/details?id=com.macdom.ble.blescanner
https://itunes.apple.com/us/app/id1221763603
